XamarinCommunityToolkit/Views/CameraView/CameraView.shared.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.ComponentModel;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class CameraView : View
+	{
+		public event EventHandler<MediaCapturedEventArgs> MediaCaptured;
+
+		public event EventHandler<string> MediaCaptureFailed;
+
+		public event EventHandler<bool> OnAvailable;
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public event EventHandler ShutterClicked;
```

Comment:
We can discuss this more and made changes in my PR... Since everything is inside the same assembly, we can change all these `EditorBrowsableState.Never` to `internal`.

I like this way because it's easy to hack, but I think the main idea to hide this is to protect developers from them selfs, so what do you think people?
A - Change to `internal`
B - Don't change at all.

Reply:
Please mark people that should be part of this discussion

Reply:
Aaah right. Yeah, in Forms those are separate libraries. You're totally right. I hate doing these things. I was too much mindlessly copy/pasting these things ;) I'll fix that. Thanks

Reply:
Changed these to be internal!